### PR TITLE
Fix the testing project and allow Core to compile

### PR DESCRIPTION
**Pull request by ilhs1096069**
* Replaced the old testing with a project (.net 7.0) with one that's on the right version (.net framework 4.8)
* Removed `StoredHighScore` from being in the csproj to compile as it does not exist anymore

:)

### CODE REVIEW

File: Core/WumpusTesting/UnitTest1.cs
Comment (roguefirework) [inline Core/WumpusTesting/UnitTest1.cs:1]: Maybe delete this?

**Review by roguefirework — CHANGES_REQUESTED** [review]

File: Core/WumpusTesting/UnitTest1.cs
Comment (ilhs1096069) [reply Core/WumpusTesting/UnitTest1.cs:1]: fixed

**Review by 1118422ILHS — APPROVED** [review]
Please don't make silly mistakes like this again

File: Core/WumpusTesting/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("WumpusTesting")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Northshore School District No.417")]
```
[the remark this answers is not in the document]
Comment (roguefirework) [reply Core/WumpusTesting/Properties/AssemblyInfo.cs:8]: Real

**Review by roguefirework — APPROVED** [review]
It may in fact look good to someone whos name may be me

**Pull request merged**